lib/Config.ts

```diff
@@ -84,6 +86,13 @@ class Config {
       disable: true,
       port: 8080,
     };
+    this.thresholds = {
+      disable: true,
+      disablefor: [],
+      maxprice: 10,
+      minprice: 0,
+      minquantity: 0.1,
```

Comment:
We're representing quantity with integer satoshis internally. `0.1` whole units seems very high for early testing phases, even for production trading it's common to see orders smaller than that, so maybe this should be like 100 or 1000 if enabled.